sopel/coretasks.py

```diff
@@ -764,11 +768,13 @@ def _record_who(bot, channel, user, host, nick, account=None, away=None, modes=N
     user.away = away
     priv = 0
     if modes:
-        mapping = {'+': sopel.module.VOICE,
-           '%': sopel.module.HALFOP,
-           '@': sopel.module.OP,
-           '&': sopel.module.ADMIN,
-           '~': sopel.module.OWNER}
+        mapping = {
+            "+": sopel.module.VOICE,
```

Comment:
Why do we change `'+'` to `"+"` already?

Reply:
Requoting is an "I'm touching it, and where it wouldn't necessitate escaping, double quotes are the eventual direction for consistency."

Reply:
Meh. I don't know who said that, but I slightly disagree. Whatever, not important.